deployment/network-operator/README.md

```diff
@@ -216,6 +216,20 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.3` | IPAM CNI Plugin image version  |
 
+## Helm Tests
+
+Network Operator has helm tests to verify deployment, currently supported tests:
+- Device Plugin Resource: This test create a pod that requests the first resource in `devicePlugin.resources`
+- RDMA Traffic: This test creates a pod that run server and client with `rping`
```

Comment:
nit

> This test creates a pod that run server and client with `rping`

This test creates a pod that test loopback RDMA traffic with `rping`

Reply:
changed